src/pat/tinymce/tinymce--implementation.js

```diff
@@ -219,6 +219,24 @@ export default class TinyMCE {
                 self.tiny = editor;
             }
         };
+        tinyOptions["setup"] = (editor) => {
+            editor.ui.registry.addMenuButton('inserttable', {
+                icon: 'table',
+                tooltip: 'Insert table 2',
```

Comment:
Can we have a better title, maybe "Insert table with table headers" ?
We should also add the option to translate this with the i18n helpers.